src/components/prompts/ReactRouterConfirmLeave.js

```diff
@@ -2,26 +2,26 @@ import React from 'react';
 import { Prompt } from 'react-router-dom';
 
 /**
- * Customize React Router on leave prompt to use <OnLeavePrompt>
+ * Customize React Router on leave/unmount prompt to use <OnLeavePrompt>
```

Comment:
Not sure what `<ReactRouter.Prompt>` has to do with unmounting.

Reply:
I thought this whole thing was triggered when `<VisitPage>` is about to unmount. Isn't that what 'leaving' is in React?

Reply:
Ah. I used "leave" as a shortcut for "navigate away from". Just saying `<ReactRouter.Prompt>` would have probably been better in the first place.